src/controllers/user.controller.ts

```diff
@@ -189,3 +189,47 @@ export const patchMarkUserAdminHandler = async (
 		});
 	}
 };
+
+/**
+ * This controller will unscribe user marketing mail.
+ *
+ * @param req request
+ * @param res response
+ *
+ * @author arbaz23
+ */
+export const updateUnsubcribeUser = async (
+	req: Request<PatchUserSchema["params"]>,
+	res: Response,
+) => {
+	const { id } = req.params;
```

Comment:
You don't need to extract id from `req.params` using the middleware https://github.com/MultiEmail/backend/blob/e1adbb9e341189cde8cabfc8e17791650a3338b9/src/middleware/deserializeUser.middleware.ts#L28

you can get id as `res.locals.user.id`

Reply:
@Arbaz23 please address this